app/src/main/java/org/mozilla/focus/browser/cursor/CursorView.kt

```diff
@@ -58,11 +63,13 @@ class CursorView(context: Context, attrs: AttributeSet) : View(context, attrs) {
     override fun onDraw(canvas: Canvas) {
         super.onDraw(canvas)
 
-        // We draw the cursor in the top-left corner, offset so the circle isn't clipped by its
-        // parent. The translationX/Y properties will draw it into the desired position on the
-        // screen. We do it this way to avoid allocating new RadialGradients, which are
-        // constructed in relation to the current position of the circle.
-        canvas.drawCircle(radius, radius, radius, paint)
+        // We have to paint a background for the shadow to apepar
+        canvas.drawRect(0f, 0f, width + 0f, height + 0f, paint)
```

Comment:
nit: let's rename paint to make it apparent how it's being used (with a transparent color).